tsconfig.json

```diff
@@ -6,7 +6,7 @@
     "sourceMap": false,
     "declaration": true,
     "removeComments": false,
-    "noImplicitAny": false,
```

Comment:
Why this change? I feel like we might want to be more lenient about this unless we have to be strict.